docs/source/index.rst

```diff
@@ -405,7 +405,8 @@ Polling is not subject to handle state with HTTP status codes (e.g. 304)
    :<json string cluster_network: (required) The cluster-only network
    :<json boolean redhat_storage: (optional) Use the downstream version of
                                   RedHat Storage.
-   :<json array monitor_hosts: (required) The monitors for the current cluster
+   :<json array monitors: (required) The monitors for the cluster you want to add this OSD to.
```

Comment:
we need to find a way to further describe these because these are useful for simple key/value stuff, not an array of JSON objects